services/src/projects/project.rs

```diff
@@ -459,24 +460,15 @@ impl Default for ProjectFilter {
     }
 }))]
 pub struct CreateProject {
+    #[validate(length(min = 1))]
+    #[validate(length(max = 256))]
     pub name: String,
     pub description: String,
```

Comment:
You previously checked for the description of not being empty, but you do not check this right now.

Reply:
5cb7a0f1d944f31e6f219bd04f3ecf3115f4280a